server/src/main/kotlin/com/xebia/functional/xef/server/Server.kt

```diff
@@ -59,7 +59,7 @@ object Server {
             requestTimeout = 0 // disabled
           }
           install(Auth)
-          install(Logging) { level = LogLevel.INFO }
+          install(Logging) { level = LogLevel.ALL }
```

Comment:
This is only needed for development, in `main` this setting should be set to INFO